### ISSUE TITLE
api.tx.sudo.sudo(api.tx.pallet.method) return 'badOrigin'

### ISSUE THREAD

**Issue by AmazingBeerBelly** [polkadot-js/api#3388]
I use the sudo module to call the pallet method, api.tx.sudo.sudo(api.tx.pallet.method(params)) In this way, I use Alice's account to signAndSend, and I get the reporting 'badOrigin'. 

but it is success to call api.tx.sudo.sudo(api.tx.balances.setBalance(ADDR, 12345, 678).

Why? Thanks a lot!

**Comment by jacogr** [comment]
Not sure. The origins checks are more st probably different, I.e. setBalance does need a root origin in the Rust code.

**Comment by joelamouche** [comment]
We (Moonbeam) have tested this locally and it seems to be a substrate problem. What parachain were you on @AmazingBeerBelly ?
cc @JoshOrndorff

**Comment by JoshOrndorff** [comment]
@AmazingBeerBelly if you want to be able to call your extrinsic with sudo, it needs to ensure that root called it, [like in `set_balance`](https://github.com/paritytech/substrate/blob/master/frame/balances/src/lib.rs#L296), or not check the origin at all. Share the code of the extrinsic you were trying to call and I can help you.

@joelamouche Having written that out, I realized something silly I did. I was trying to submit a system::remark with sudo. When in fact system::remark is _not_ meant for root for exactly the reason I just explained above. I confirmed that sudo is indeed working on Moonbeam just now by submitting balances::setBalance. So it is still a mystery why we can't submit evm transactions via sudo, but it is not related to Polkadot JS.

Sorry to scribble on your repo Jaco. Hopefully helping @AmazingBeerBelly makes up for it partially.
